### PR TITLE
store/tikv: open switch of topn.

### PR DESCRIPTION
**Pull request by hanfei1991**
implement topn for mock-tikv.
@shenli @coocood @zimulala @tiancaiamao PTAL

### CODE REVIEW

File: store/tikv/mock-tikv/cop_handler.go [hunk 1]
```diff
@@ -221,7 +261,7 @@ func (h *rpcHandler) getChunksFromSelectReq(ctx *selectContext) ([]tipb.Chunk, e
 		if limit == 0 {
 			break
 		}
-		chunks, err = h.getRowsFromRange(ctx, ran, &limit, desc, chunks)
+		chunks, err = h.getRowsFromRange(ctx, ran, &limit, ctx.descScan, chunks)
```
Comment (coocood) [inline store/tikv/mock-tikv/cop_handler.go]: parameter `ctx.desScan` can be removed.

File: store/tikv/mock-tikv/cop_handler.go [hunk 2]
```diff
@@ -487,7 +526,7 @@ func (h *rpcHandler) getChunksFromIndexReq(ctx *selectContext) ([]tipb.Chunk, er
 		if limit == 0 {
 			break
 		}
-		chunks, err = h.getIndexRowFromRange(ctx, ran, desc, &limit, chunks)
+		chunks, err = h.getIndexRowFromRange(ctx, ran, ctx.descScan, &limit, chunks)
```
Comment (coocood) [inline store/tikv/mock-tikv/cop_handler.go]: ditto

**Comment by shenli** [comment]
LGTM

**Comment by hanfei1991** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**